interfaces/builtin/modem_manager.go

```diff
@@ -199,6 +202,9 @@ dbus (send)
     path=/org/freedesktop/ModemManager1{,/**}
     interface=org.freedesktop.DBus.Properties
     member="Get{,All}",
+
+# allow communicating with mbim-proxy server
+unix (connect, receive, send) type=stream peer=(addr="@{mbim,qmi}-proxy"),
```

Comment:
Shouldn't the classic variant have `peer=(label=unconfined,addr=...)` if we are to allow talking to modem manager provided by the classic host or maybe that's too strict?

Reply:
I agree it would be good to try and scope this for the classic case but I think overall that would complicate this interface too much to try and specify an extra `label=unconfined` for just classic. Also I don't see much risk in *not* specifying this in the classic case since what is the intended threat model? Having one snap create a mbim/qmi-proxy socket and hence this rule would then allow another snap which plugs `modem-manager` to talk to that snap? I don't see that as a huge risk, plus I wonder if by specifying `unconfined` that would stop a snap from talking to an intentionally confined `mbim-proxy` etc (ie. if someone were to write an AppArmor profile for `mbim-proxy` or we decided to ship one in the future). As such I don't think this is needed but it is a good suggestion.